pywinauto/unittests/test_uiawrapper.py

```diff
@@ -1419,6 +1420,50 @@ def test_button_click(self):
             itm = lst_ctl.children()[1]
             self.assertEqual(itm.texts()[0], u'Red')
 
+    class ToolbarMfcTests(unittest.TestCase):
+
+        """Unit tests for ToolbarWrapper class on MCF demo"""
+
+        def setUp(self):
+            """Set some data and ensure the application is in the state we want"""
+            _set_timings()
+
+            # start the application
+            self.app = Application(backend='uia')
+            self.app = self.app.start(mfc_app_rebar_test)
```

Comment:
2 lines could be collapsed to one liner: `self.app = Application(backend='uia').start(mfc_app_rebar_test)`